vpr/src/util/mmap_file.cpp

```diff
@@ -0,0 +1,35 @@
+#include "mmap_file.h"
+#include "vpr_error.h"
+#include "vtr_log.h"
+
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#include "kj/filesystem.h"
```

Comment:
What is kj? Another library?

How portable is this code?

Reply:
kj is capnproto's OS abstraction layer.  It is not required to be used in VPR, but it provide an OS-independent way to mmap the file for reading.